rs/http_endpoints/public/src/call.rs

```diff
@@ -289,7 +289,7 @@ impl IngressMessageSubmitter {
 
     /// Attempts to submit the ingress message to the ingress pool.
     /// An [`HttpError`] is returned if P2P is not running.
-    pub(crate) fn try_submit(self) -> Result<(), HttpError> {
```

Comment:
Why rename?

Reply:
looking at the implementation there is not much trying :) the operation will always succeed